settings.gradle.kts

```diff
@@ -1,12 +1,25 @@
-rootProject.name = "lightning-kmp"
-
 pluginManagement {
     repositories {
         gradlePluginPortal()
         maven("https://dl.bintray.com/kotlin/kotlin-eap")
     }
 }
 
-include(
-    ":PhoenixCrypto"
-)
+dependencyResolutionManagement {
+    repositories {
+        // using the local maven repository with Kotlin Multi Platform can lead to build errors that are hard to diagnose.
+        // uncomment this only if you need to experiment with snapshot dependencies that have not yet be published.
+        // mavenLocal()
+        maven("https://oss.sonatype.org/content/repositories/snapshots")
+        mavenCentral()
+        google()
+    }
+}
+
+rootProject.name = "lightning-kmp"
+
+include(":lightning-kmp-ios-crypto")
+include(":lightning-kmp-core")
+
+project(":lightning-kmp-ios-crypto").projectDir = file("./modules/ios-crypto")
+project(":lightning-kmp-core").projectDir = file("./modules/core")
```

Comment:
The projects' name `lightning-kmp-xxx` is quite large, it makes running specific gradle commands a bit annoying (for example, `./gradlew :lightning-kmp-core:publishToMavenLocal`). Why not use `core` and `ios-crypto` as names ?

Reply:
Done in 684272ed. I didn't know that I could override the artifact id.

Reply:
This has side effects on publication, had to revert changes...